bundles/io/org.eclipse.smarthome.io.transport.dbus/build.properties

```diff
@@ -0,0 +1,9 @@
+bin.includes = META-INF/,\
+               OSGI-INF/,\
+               .,\
+               about.html,\
+               lib/libdbus-java-2.7.jar,\
+               lib/unix-0.5.jar

```

Comment:
>  I just wondered if we should check this?

Well, it could be worth a try, yes. I will create a CQ for dbus-java and ask at the same time if they could ask the author for the other lib as well. Do you possibly have a link to the official sources?

> this will be dependant on the system which is why the library is NOT in the bundle.

Well, I found the file in the root of the bundle, that's why I was asking about it :-)
But from your answer, I suppose that we will not have to bundle it as it is automatically found on the system library path of any Linux system, correct?